demisto_sdk/commands/upload/uploader.py

```diff
@@ -421,6 +428,29 @@ def print_summary(self) -> None:
 
             logger.info(f"[green]SUCCESSFUL UPLOADS:\n{uploaded_str}\n[/green]")
 
+        if self._skipped_upload_marketplace_mismatch:
+            marketplace_mismatch_str = tabulate(
+                (
+                    (
+                        item.path.name,
+                        item.content_type,
+                        self.marketplace,
+                        item.marketplaces,
+                    )
+                    for item in self._skipped_upload_marketplace_mismatch
+                ),
+                headers=[
+                    "NAME",
+                    "TYPE",
+                    "MARKETPLACE",
+                    "FILE_MARKETPLACES",
+                ],
+                tablefmt="fancy_grid",
+            )
+            logger.info(
+                f"[yellow]SKIPPED UPLOADED DUE TO MARKETPLACE MISMATCH:\n{marketplace_mismatch_str}\n[/yellow]"
```

Comment:
```suggestion
                f"SKIPPED UPLOADED DUE TO MARKETPLACE MISMATCH:\n{marketplace_mismatch_str}]"
```
would it print in yellow without the marks?